app/components/channel_icon/index.tsx

```diff
@@ -115,30 +115,49 @@ const ChannelIcon = ({
         activeGroup = styles.groupActive;
     }
 
-    if (isInfo) {
-        activeIcon = isUnread && !isMuted ? styles.iconInfoUnread : styles.iconInfo;
-        activeGroupBox = styles.groupBoxInfo;
-        activeGroup = isUnread ? styles.groupInfoUnread : styles.groupInfo;
+    if (isOnCenterBg) {
+        activeIcon = isUnread && !isMuted ? styles.iconUnreadOnCenterBg : styles.iconOnCenterBg;
+        activeGroupBox = styles.groupBoxOnCenterBg;
+        activeGroup = isUnread ? styles.groupUnreadOnCenterBg : styles.groupOnCenterBg;
     }
 
     if (isMuted) {
         mutedStyle = styles.muted;
     }
 
-    let icon;
+    const commonStyles = [
+        style,
+        mutedStyle,
+    ];
+
+    const commonIconStyles = [
+        styles.icon,
+        unreadIcon,
+        activeIcon,
+        commonStyles,
+        {fontSize: size},
+    ];
+
+    let icon = null;
     if (isArchived) {
         icon = (
             <CompassIcon
                 name='archive-outline'
-                style={[styles.icon, unreadIcon, activeIcon, {fontSize: size, left: 1}]}
+                style={[
```

Comment:
In webapp we have deprecated use of compass icons should we do the same here as well?

Reply:
no, here the SVG version is much much heavier than using the font directly.